index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header_nav">
+        <div class="logo black">
+          <a class="black" href="#">Logo</a>
+        </div>
+        <div class="header_menu">
+          <img class="menu_icon" src="assets/ic_menu.png" alt="menu" />
+        </div>
+      </nav>
+    </header>
+    <section class="headline">
+      <h1 class="headline_title">
+        <span class="sub-headline_name">Hey there, I'm Arslan</span>
+        <span class="sub-headline">I'm a software developer</span>
+      </h1>
+      <p class="headline_content">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don,t hestiate to contact me.
+      </p>
+      <div class="backimg"></div>
+      <div class="addline_social">
+        <ul class="social_links">
+          <li class="social_link">
+            <a href="#">
+              <img
+                class="social_icon"
+                src="assets/ic_twitter.svg"
+                alt="Twitter"
+              />
+            </a>
+          </li>
+          <li class="social_links"></li>
+          <li class="social_link">
+            <a href="#">
+              <img
+                class="social_icon"
+                src="assets/ic_linkedIn.svg"
+                alt="LinkedIn"
+              />
+            </a>
+          </li>
+          <li class="social_link">
+            <a href="#">
+              <img
+                class="social_icon"
+                src="assets/ic_github.svg"
+                alt="Github"
+              />
+            </a>
+          </li>
+          <li class="social_link">
+            <a href="#">
+              <img
+                class="social_icon"
+                src="assets/ic_medium.svg"
+                alt="Medium"
+              />
+            </a>
+          </li>
+          <li class="social_link">
+            <a href="#">
+              <img
+                class="social_icon"
+                src="assets/ic_angellist.svg"
+                alt="AngelList"
+              />
+            </a>
+          </li>
+        </ul>
+      </div>
```

Comment:
I suggest you follow the Figma template and use the correct space from the headline content and the social links. Kindly take a look at the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=CTghrk2ad5bwG4yo-0) to make your project fit the template.

> ### The Figma Desgin
>
><img width="447" alt="Screenshot 2023-04-28 at 12 11 33" src="https://user-images.githubusercontent.com/91237525/235199407-3404deca-8998-40c9-9b4c-1908fd54297d.png">
>
>
>### Your Design
>
><img width="327" alt="Screenshot 2023-04-28 at 12 04 58" src="https://user-images.githubusercontent.com/91237525/235199133-0385c980-1d62-4366-afa7-543c6e2863f6.png">
>
>
>
>
> Kindly I suggest you add this button bellow as it is in the Figma template
>
>
><img width="441" alt="Screenshot 2023-04-28 at 12 14 19" src="https://user-images.githubusercontent.com/91237525/235199935-00514efa-1a50-4499-a84a-99e9baab2b4f.png">